File: scienceCenterLuka/src/main/java/com/project/dto/UpdateArticleChangesDto.java
package com.project.dto;

import java.io.Serializable;
import java.util.List;

import org.camunda.bpm.engine.form.FormField;

import com.project.model.OpinionAboutArticle;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UpdateArticleChangesDto implements Serializable{/**
	 * 
	 */
	private static final long serialVersionUID = 7267623785811514588L;
	
	private NewArticleRequestDto newArticleRequestDto;
	private NewArticleResponseDto newAarticleResponseDto;
	
	private List<OpinionAboutArticle> reviewersOpinion;
	private List<OpinionAboutArticle> editorsOpinion;
	
	private OpinionAboutArticle authorsMessage;
	
	private List<FormField> fields;
	private List<FormSubmissionDto> fieldResults;
}

File: scienceCenterLukaFrontend/src/app/model/update-article-changes-dto.ts
import { OpinionAboutArticle } from './opinion-about-article';
import { NewArticle } from './new-article';

export class UpdateArticleChanges {

     newArticleRequestDto;   
     newAarticleResponseDto: NewArticle;
                                                 
     reviewersOpinion: OpinionAboutArticle[];  
     editorsOpinion: OpinionAboutArticle[];    
                                                 
    authorsMessage: OpinionAboutArticle;    
    
    fields: any[] = [];
    fieldResults: any[] = [];

    constructor() {}
}